public/css/leaderboard.css

```diff
@@ -0,0 +1,61 @@
+
+div.podium {
+    display: inline-block;
+    width: auto;
+    text-align: center;
+    padding: 35px;    
+}
+div.podium p {
+    margin-top: 0;
+    color: #000000;
+    font-size: 20px;
+}
+div.podium p span.score {
+    font-weight: bold;
+}
+
+ol, li {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+ol {
+    display: flex;
+}
+
+li {
+    display: inline-block;
+}
+
+li.silver {
+    order: -1;
+}
+
+.silver, .bronze {
+    transform: scale(0.75);
+}
+
+li img {
+    display: inline-flex;
+}
+
+li span.score {
+    position: relative;
+    top: 15px;
+    border: 1px solid black;
+    padding: 12.5px;
+    font-size: 25px;
+}
+
+li.gold .score {
+    background-color: rgba(255,215,0,0.7);
+}
+
+li.silver .score {
+    background-color: rgba(192,192,192,0.7);
+}
+
+li.bronze .score {
```

Comment:
can we use more proper pictures please

Reply:
can we use the logo until profile pictures are done?

Reply:
I'll change it to the logo :cry:

Reply:
why are we using profile pictures here?